client/lib/localforage/index.js

```diff
@@ -27,6 +27,7 @@ const config = {
 };
 
 let _ready = false;
+
```

Comment:
was this accidentally included?

Reply:
It looks intentional to add some breathing room, but as it is now the only change to this file, let's remove it.

Reply:
I think it was caused by first making changes to this module in one commit and then reverting them in a later one. The revert was not 100% precise and this +1 line remained.